### PR TITLE
added routes via resources

### PR DESCRIPTION
**Pull request by henderson907**
As far as I could think we didn't need to nest anything inside the resources :venues, but now I'm wondering if we need to do that for the reviews or if it's going to be within the venues controller?

### CODE REVIEW

File: app/controllers/pages_controller.rb [hunk 1]
```diff
@@ -3,4 +3,10 @@ class PagesController < ApplicationController
 
   def home
   end
+
+  def about
+  end
+
+  def my_profile
```
Comment (pmunozs) [inline app/controllers/pages_controller.rb:10]: Wouldn't this be part of the user?

File: config/routes.rb [hunk 2]
```diff
@@ -1,8 +1,9 @@
...
   devise_for :users
   root to: "pages#home"
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
-  # Defines the root path route ("/")
-  # root "articles#index"
+  get "about", to: "pages#about"
+  get "my_profile", to: "pages#my_profile"
+
+  resources :venues
```
Comment (pmunozs) [inline config/routes.rb:8]: And yes, as you mentioned I think reviews would be nested

File: app/controllers/pages_controller.rb (hunk 1 above)
Comment (henderson907) [reply app/controllers/pages_controller.rb:10]: Yep I think you're right, I completely forgot about devise.

File: config/routes.rb (hunk 2 above)
Comment (henderson907) [reply config/routes.rb:8]: on it

File: config/routes.rb [hunk 3]
```diff
@@ -1,8 +1,11 @@
 Rails.application.routes.draw do
   devise_for :users
   root to: "pages#home"
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
-  # Defines the root path route ("/")
-  # root "articles#index"
+  get "about", to: "pages#about"
+  get "my_profile", to: "pages#my_profile"
```
Comment (pmunozs) [inline config/routes.rb:6]: You can remove this line. This action will be inside the users:
users#my_profile

**Review by pmunozs — APPROVED** [review]

**Pull request merged**